purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -175,7 +173,6 @@ fun Purchases.purchaseSubscriptionOptionWith(
 /**
  * Purchase a subscription [StoreProduct]'s [PurchaseOption], upgrading from an old product.
  * @param [activity] Current activity
- * @param [storeProduct] The storeProduct of the product you wish to purchase
  * @param [purchaseOption] Your choice of purchase options available for the subscription StoreProduct
```

Comment:
what do you think about changing these docs from `the subscription StoreProduct` -> `a subscription StoreProduct`.

Reply:
I love that, TBH! `the` doesn't feel right 😊